pkgs/development/interpreters/erlang/generic-builder.nix

```
@@ -113,10 +125,14 @@ stdenv.mkDerivation ({
   enableParallelBuilding = parallelBuild;
 
   postPatch = ''
-    patchShebangs make
+    substituteInPlace lib/wx/c_src/wxe_impl.cpp \
+      --replace 'temp > NULL' 'temp != NULL'
```

```suggestion
      --replace 'temp > NULL' 'temp != NULL'
```
Please use `"` instead of `'`.

---

Use `--replace-fail` as well.